docs/plugins/_layer_data_guide.md

```diff
@@ -3,16 +3,16 @@
 
 When transfering data to and from plugins, napari does not pass `Layer` objects
 directly. Instead, it passes (mostly) pure-python and array-like types,
-deconstructed into a {class}`tuple` that we refer to as a `LayerData` tuple.  This type shows
-up often in plugins and is explained here.
+deconstructed into a {class}`typing.NamedTuple` that we refer to as a `LayerData` tuple.
+This type shows up often in plugins and is explained here.
 
 ### Informal description
 
 ```py
 (data, [attributes, [layer_type]])
 ```
 
-A `LayerData` tuple is a tuple of length 1, 2, or 3 whose items, in order, are:
+A `LayerData` tuple is a namedtuple of length 1, 2, or 3 whose items, in order, are:
```

Comment:
We can't go this far... since it implies that *all* LayerData tuples are named tuples.  It's fine to create a named tuple convenience if you or others want to use it, but ultimately the whole reason for using a tuple instead of a napari object (like a Layer) is so that no one has to import napari in a plugin just to return a "napari specific" object.  So, a `LayerData` tuple _must_ remain, more generally, just a plain `tuple`.  A named tuple is a tuple subclass, but not vice versa.  So these docs have to remain as a plain tuple (or technically, anyone _not_ returning a named tuple has now broken our API)